### ISSUE TITLE
Alright then, first question (sidechain + meters)

### ISSUE THREAD

**Issue by BrootLee** [swh/ladspa#43]
Hi.
I use the SC compressors on Ardour4. SC2 and SC3 have sidechain but not meters, while SC4 and SC4 mono have meters but not sidechain. How to get the best out of both worlds? e.g. meters + sidechain
thanks

**Comment by swh** [comment]
I guess you'd have to get a C programmer to contribute an SC5 with both? The XML for SC4 would have to be copied as a starting point.

It's not really hard to do, but I don't have a machine that can build LADSPA code at the moment.

**Comment by BrootLee** [comment]
Thanks for clarifying that.
In the meantime, isn't there a smart/hackish way to hook two of them up (e.g. sc2/3 plus sc4) to make them work as one and get both fearures?

**Comment by swh** [comment]
I don't think so, if you stack them in series the effects will combine. You could maybe run them in parallel, and take the audio output from the side chained one? The metered output wouldn't be quite right. Not sure which hosts would let you do that though.

**Comment by BrootLee** [comment]
I believe Ardour5 now has the ability to customize the internal wiring of plugins, but not sure up to what extent. I'll experiment when I update Ardour
